### ISSUE TITLE
Use elementary as icon theme fallback

### ISSUE THREAD

**Issue by NicoHood** [horst3180/arc-icon-theme#9]
I **really** like this theme. But as it is incomplete you really need a **beautiful** fall back theme. I also suggest to add the `elementary` theme as fall back, as it perfectly suits the style.

```
sed -ie "s/Inherits=.*/Inherits=Moka,Faba,elementary,Adwaita,gnome,hicolor/" Arc/index.theme
```

The elementary theme can be downloaded from the official arch repo:
https://www.archlinux.org/packages/community/any/elementary-icon-theme/

I created PR #12 to add elementary as 2nd fallback.
The advantage of elementary is that is is a) officially supported and b) has no application icons but resolves most of the missing other icons.

More Links:
- https://github.com/elementary/icons
- https://launchpad.net/elementaryicons
- https://github.com/mank319/elementaryPlus
- https://aur.archlinux.org/packages/elementary-xfce-icons/

**Comment by NicoHood** [comment]
It would also make sense to use Faba as fallback between moka and elementary, as faba has no application icons (like firefox etc) but still adds some of the missing icons. Thats still (for me) a better alternative than moka.

**Comment by ghost** [comment]
Moka is far more complete than Elementary Icon Theme, that only includes icons for all Elementary applications and for some GNOME applications.

Using Elementary's as fallback isn't a improvement, Elementary Icons can be beautiful, but the "completness" is the target of taking Moka as default fallback.

**Comment by NicoHood** [comment]
Yes I see. However I was searching for something which still looks good for the few missing icons (Terminal, File Manager, Settings) which is in the official repos if possible and does not include Application icons (as I do not want different Application icons except for terminal).

Thatswhy I added Faba here. But I still have to play with this some more. Other suggestions also appreciated. I also hope that the few missing Icons get replaced here in the Arc theme, so you no longer need a fallback theme at all.

**Comment by moriel5** [comment]
I recommend using Faba as the fallback theme, now that's a definitely good theme.
Plus, Faba's fallback theme is Gnome, so there is no shortage of icons.

And not only Thunar is missing a default icon, also Nautilus is missing one.

**Comment by maeslor** [comment]
I think it should be better to include or substitute elementary as fallback theme for [elementary-xfce](https://github.com/shimmerproject/elementary-xfce), as it contains more icons beyond pantheon/elementary-os contexts. Elementary Xfce also supports some Gnome apps beyond Xfce.
